v3-sdk/swap-and-add-liquidity/src/libs/conversion.ts

```diff
@@ -0,0 +1,6 @@
+export function fromReadableAmount(amount: number, decimals: number): number {
+  return amount * Math.pow(10, decimals)
+}
+export function toReadableAmount(rawAmount: number, decimals: number): string {
+  return (rawAmount / Math.pow(10, decimals)).toString()
+}
```

Comment:
Overflow safe ones?

Reply:
Not even the balance works with the safe ones 🤔 (for pretty much all of them)